src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -252,6 +254,11 @@
             .set(DATA)
             .build();
 
+    private final OctetsFW nullConstructor = new OctetsFW.Builder()
+        .wrap(new UnsafeBuffer(new byte[1]), 0, 1)
+        .set(new byte[] {0x40})
+        .build();
+
```

Comment:
This can be simplified to a readonly wrap (instead of an allocate + copy) via:
```
    private final OctetsFW nullConstructor = new OctetsFW()
        .wrap(new UnsafeBuffer(new byte[] {0x40}), 0, 1)
        .build();
```

Reply:
When I changed the code to the suggested way, I get an empty octet? (`oxtets[0]`)

Reply:
```
    public static void main(String[] args)
    {
        OctetsFW nullConstructor = new OctetsFW()
                .wrap(new UnsafeBuffer(new byte[] {0x40}), 0, 1);

        System.out.println(nullConstructor);
    }
```
outputs
```
octets[1]
```